app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const ContactsMenuBar = ({ showMenu }) => {
+  const [isSegmentsSearchCollapsed, setIsSegmentsSearchCollapsed] =
+    useState(true);
+  const [isTagsSearchCollapsed, setIsTagsSearchCollapsed] = useState(true);
+
+  return (
+    <div className="flex">
+      <MenuBar showMenu={showMenu} title="Notes">
```

Comment:
Why do we need this `div` here? Why not pass in these classNames as part of the MenuBar itself?